src/components/sidebar/index.js

```diff
@@ -81,6 +82,15 @@ const Sidebar = ({}) => {
     fetchStore()
   }, [])
 
+  const triggerHandler = () => {
+    const id = triggerHandler.id++
+    return {
+      open: currentlyOpen === id,
+      handleTriggerClick: () => setCurrentlyOpen(id),
+    }
+  }
+  triggerHandler.id = 0
```

Comment:
What does this line do? Would be great to move this to a place, that makes a bit more sense or add a comment :)

Reply:
@sebastiannicolajsen